Yosemite/Yosemite/Internal/String+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension String {
+
+    private static let slugSafeCharacters = CharacterSet(charactersIn: "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-")
+
+    public var slugified: String? {
```

Comment:
I believe this can be module-level only. 

```suggestion
    var slugified: String? {
```